projects/lxc/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/lxc/lxc"
+language: c
+primary_contact: ""
```

Comment:
@brauner I forgot to ask who OSS-Fuzz should send bug reports to: https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary. Unfortunately, it only works with gmail addresses: https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication

Reply:
Ah, I think:
christian.brauner@ubuntu.com
stgraber@ubuntu.com
they are both aliases and should be gmail. If that doesn't work:
stephane.graber@canonical.com
christian.brauner@canonical.com